pyproject.toml

```diff
@@ -28,6 +29,7 @@ boto3 = "1.35.48"
 cryptography = "43.0.3"
 psycopg = {version = "3.2.3", extras = ["binary"]}
 yaptide-converter = { git = "https://github.com/yaptide/converter.git", rev = "a9753a4f7856df868fc1c4f80a3c7e77622f9bb9"}
+psycopg2-binary = "^2.9.10"
```

Comment:
I don't like having both `psycopg` with binaries extras and `psycopg2-binary` defined separately. Do we really need the `psycopg2-binary` here ?

Reply:
Not to mention that according to official docs from https://www.psycopg.org/docs/install.html#quick-install

<img width="734" alt="Screenshot 2024-10-25 at 15 10 35" src="https://github.com/user-attachments/assets/1c1c487a-7a12-4eb3-a5ff-0ece90f433e6">

Reply:
I had some errors at beginning of work with flask-migrate so I installed it. But now i checked on fresh branch without psycopg2 and it's ok.